ansible_runner/utils/__init__.py

```diff
@@ -198,11 +198,12 @@ def dump_artifacts(kwargs):
                 kwargs['inventory'] = dump_artifact(obj, path, 'hosts')
 
     for key in ('envvars', 'extravars', 'passwords', 'settings'):
-        obj = kwargs.get(key)
-        if obj and not os.path.exists(os.path.join(private_data_dir, 'env', key)):
-            path = os.path.join(private_data_dir, 'env')
-            dump_artifact(json.dumps(obj), path, key)
-            kwargs.pop(key)
+        if key == 'passwords' and kwargs.get('store_passwords', True) or key != 'passwords':
```

Comment:
why just passwords as opposed to all of these, envvars/extravars/passwords/settings?

I think a similar security argument can apply to all of these, and I don't know that functionality would be harmed by not dumping any of them. So I'm interested to ask if we could have 1 configurable that will tell it to not dump _any_ of these. So instead of `store_passwords`, something like `store_inputs`, name a WIP.